pipeline/merge_pipeline.py

```diff
@@ -89,7 +89,8 @@ def create_queries(args, start_date, end_date):
         yield query
         start_window = end_window + datetime.timedelta(days=1)
 
-
+def filter_valid_coordinates(obj):
+    return obj['mean_latitude'] >= -89.99 and obj['mean_latitude'] <= 89.99
 
```

Comment:
I think would be slightly clearer as:
`return -89.99 <= obj['mean_latitude'] <= 89.99` , but no strong preference.

Reply:
Definitely, thanks!